### ISSUE TITLE
Jenny's Choice

### ISSUE THREAD

**Issue by brightbold** [Inexorable-Fate/mpc_projects#133]
### Project Name

Jenny's Choice

### Author(s)

Seth Oakman

### Has the project been verified by the creator as finalized and ready for print and play?

- [X] Yes
- [ ] No
- [ ] Not Sure

### Link to cards

https://drive.google.com/drive/folders/1OsnD6w8zdDVCBUjrl6ibQApgbHCJGTP6?usp=drive_link

### Link to a web page with more details about this project

https://arkhamcentral.com/index.php/jennys-choice/

### What’s included in this project?

- [ ] Campaign
- [X] Standalone scenario
- [ ] Investigator(s) & signatures
- [ ] Player Cards
- [ ] Add-Ons (utility cards and other extras that don’t fall into one of the above categories)

### Additional details

Requires Jenny Barnes investigator to play

### Contact Details

@Bonzman72 — Bonzman on MB Discord

**Comment by brightbold** [comment]
Bonzman is redoing in Arno so hold off until that's done.

**Comment by dancuadra** [comment]
@brightbold Please assign to me

**Comment by dancuadra** [comment]
Still need to add bleed to investigator mini card. Got stuck troubleshooting bleedy.py. Will come back to this.

**Comment by brightbold** [comment]
> Still need to add bleed to investigator mini card. Got stuck troubleshooting bleedy.py. Will come back to this.

If it's just one minicard, you said you have Photoshop so you will actually get better results with Generative Expand. I usually do that for minicards anyway, since MPCs minicards are wider than FFGs. So you could skip bleedy if you wanted to move ahead with this.

**Comment by dancuadra** [comment]
Yes, that was my plan for this one. Photoshop macros may even be sufficient for normal card bleed to and would be much easier than bleedy. I'll get back to it when I finish the current Strange Eons project.

**Comment by dancuadra** [comment]
@brightbold I found the instructions a bit confusing, but I think I did this correctly. Please review the attached file and the step by step work instructions ([section 4: Uploading to MPC](https://docs.google.com/document/d/1gFG7qrNIintDnv-QEHWcgLUz_3-GCO-Q5rwZNm8pWys/edit?tab=t.2v0fa0xnare3)). Thanks.

[Jenny'sChoice.json](https://github.com/user-attachments/files/18134424/Jenny.sChoice.json)

**Comment by brightbold** [comment]
This looks great! A few notes (which I also posted in DM so I'm just putting them here for posterity):

- I think you have instructions on getting the project code but you just forgot to do it for Jenny's Choice, so I added them.
- the "artist" line in the json is just for the cover art, so if we don't know who it is I just leave it blank. Since you didn't have a piece of cover art, you can definitely leave this blank. (I added one so it's good now.)
the description was pretty long so I edited it down
- Oh! Here's a thing I didn't tell you. The info line is for requirements of things you might need to already own, or any other "know before you purchase" information. So I moved the requirement that you own Jenny Barnes from the `description` to `info`.

**Comment by dancuadra** [comment]
@brightbold Can you please attach the updated .json here so I can compare with my version to highlight the changes? I want to be sure I understand the correct formatting of the project codes. Thanks.

**Comment by brightbold** [comment]
json looks good. I copied over the three changes I mentioned in my comment above.
